tools/sof_ri_info/sof_ri_info.py

```diff
@@ -31,39 +31,6 @@
 # - add a new one as array of bytes and append entry to KNOWN_KEYS below.
 #   you can use --full_bytes to dump the bytes in below format
 
-APL_INTEL_PROD_KEY = bytes([0x1f, 0xf4, 0x58, 0x74, 0x64, 0xd4, 0xae, 0x90,
```

Comment:
Please don't remove this key (and restore the older ones in different PR, sorry I missed this), this is going to break sof-bin tests for no good reason. This is just a pure constant with zero actual logic depending on it.

Reply:
@marc-hb ok, removed. It wasn't changed in earlier PRs - BYT / BDW didn't use signed firmware AFAIK